server/chat-plugins/suspect-tests.ts

```diff
@@ -1,6 +1,7 @@
 import {FS} from '../../lib/fs';
 
 const SUSPECTS_FILE = 'config/suspects.json';
+const WHITELIST = ["kris"];
```

Comment:
Do we want this to be a configuration setting rather than a hardcoded entry?

Reply:
I would prefer that, personally; that way, it's specific to each server.

Reply:
It would be nice but Zarel prefers this method

Reply:
It sucks for other servers but makes it easier for people to PR changes to the list. I'm neutral, as long as I'm not the one maintaining the configuration setting – it all depends on how frequently people make changes to the list, and how other servers do things like this.

In the past, we've also based it on "roomauth in Dev". Would that work here? Whitelist all dev roommods?

Reply:
Dev roommod is a pretty large group (annika, austin, bmelts, chaos, dawoblefet, hoeenhero, instruct, jetou, joim, kaphotics, kris, marty, mia, panpawn, peach, pyuk, sirdonovan, slayer95, theimmortal, urkerab, v4, viett, xfix). Given that we originally only permitted upperstaff to add suspects, this seems like a big jump. I think Dev roommods are generally either inactive or trustworthy, but I question why we would expose suspect management to such a large group?

In my mind, Dev roommod has been used in conjunction with other things (such as global staff or room staff), not as a primary permission check, but just to verify that the user is technically competent and won't break things. (This was proposed for Anubis to manage hosts and Mia/Kris to add regexes to the Help autoresponder.)

I can maintain the config setting if we decide to go that route.